.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,7 +2,10 @@
 <!-- Bug, Chore, Documentation, Feature -->
 
 **What this PR does/ why we need it**:
-<!-- Explain, without going into the details, what this PR does, and what problem it solves. -->
+<!--
```

Comment:
@armandgrillet I would prefer to have the hint at the very top as I did in #223. wdyt?

Reply:
ooh +1 to this suggestion